Localize/localize-pipeline.yml

```diff
@@ -2,20 +2,114 @@ name: $(Date:MMddyy)$(Rev:.rrrr)
 
 trigger: none
 
+schedules:
+- cron: 0 8 * * Mon # mm HH DD MM DW
+  displayName: Localization update
+  branches:
+    include: 
+    - Localization
+  always: true
+
 stages:
 - stage: __default
   jobs:
-  - job: Job1
+  - job: localizationUpdate
+    displayName: 'Update localization'
     pool:
       vmImage: windows-latest
     steps:
+
+    - checkout: self
+      persistCredentials: true
+
+    - powershell: |
+        npm install
+      displayName: 'npm install'
+      
+    - powershell: |
+        $sprintInfo = Invoke-WebRequest https://whatsprintis.it -Headers @{"Accept"= "application/json"} | ConvertFrom-Json
```

Comment:
I'd suggest placing scripts (even short ones) in an external ps1 instead of inline.

Advantage is the powershell can be tested independently of the pipeline, also practices can be used to avoid/prevent errors and fail early e.g. Set-StrictMode, ErrorActionPreference=stop, PSScriptAnalyzer linter

Reply:
Please consider that this is a temporary solution to complete the initial pipelines configuration - we already use the same scripts in another repos that go through the translation flow. We planning to replace these scripts later, when we complete pipeline configuration for all remaining repos.

Reply:
👍Sounds good.